pkg/services/ngalert/store/alertmanager.go

```diff
@@ -33,6 +35,23 @@ func (st *DBstore) GetLatestAlertmanagerConfiguration(query *models.GetLatestAle
 	})
 }
 
+// GetAllLatestAlertmanagerConfiguration returns the latest configuration of every organization
+func (st *DBstore) GetAllLatestAlertmanagerConfiguration() ([]*models.AlertConfiguration, error) {
+	var result []*models.AlertConfiguration
+	err := st.SQLStore.WithDbSession(context.Background(), func(sess *sqlstore.DBSession) error {
+		condition := builder.In("id", builder.Select("MAX(id)").From("alert_configuration").GroupBy("org_id"))
+		err := sess.Table("alert_configuration").Where(condition).Find(&result)
```

Comment:
This can be all on one line such as:

```
if err := sess.Table("alert_configuration").Where(condition).Find(&result); err != nil {
    return err
}
```